cmd/k8s-operator/egress-services.go

```diff
@@ -540,13 +540,13 @@ func svcNameBase(s string) string {
 	}
 }
 
-// unusedPort returns a port in range [3000 - 4000). The caller must ensure that
-// usedPorts does not contain all ports in range [3000 - 4000).
+// unusedPort returns a port in range [10000 - 20000). The caller must ensure that
```

Comment:
Maybe 1000 - 1100 or similar range, if there is someone who legitimately needs more we probably want to hear from them